saleor/checkout/migrations/0027_checkout_channel.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.0.6 on 2020-07-15 07:24
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def add_channel_slug(apps, schema_editor):
+    Channel = apps.get_model("channel", "Channel")
+    Checkout = apps.get_model("checkout", "Checkout")
+
+    if Checkout.objects.exists():
```

Comment:
Why do we need this? The iterator below won't run without if there are no checkouts.